### ISSUE TITLE
Paramiko error using community.routeros.command in playbook 

### ISSUE THREAD

**Issue by salgua** [ansible-semaphore/semaphore#852]
Hi,
I'm trying to use `community.routeros.command` in a playbook using **semaphore** on last stable version on **Ubuntu 18.04**. I always obtain this error:

    12:32:27 PM
    TASK [community.routeros.command] **********************************************
    12:32:30 PM
    fatal: [routeros-03]: FAILED! => changed=false
    12:32:30 PM
    msg: 'paramiko is not installed: No module named ''paramiko'''

I tried to install paramiko with

    apt install python3-paramiko

but it doesn't solve the issue. I installed Semaphore with snap, I think snap uses own versions of Python and Ansible. How can I solve this issue?

**Comment by fiftin** [comment]
Yes, it is true, snap uses internal version of Python and Ansible. I will try to reproduce and answer soon.

**Comment by fiftin** [comment]
Hi @salgua ,

Try next:

```
sudo -s
cd /root/snap/semaphore/common/ansible_env/bin
source ./activate
pip3 install paramiko
```

**Comment by miguletztim** [comment]
The issue persists when using the Docker image semaphore:v2.10.35. Paramiko is supposed to be bundled with Ansible-Core (as per the official documentation [here](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/paramiko_ssh_connection.html)), but there seems to be an issue preventing it from being properly included or recognized.

Note: Installing Paramiko manually via pip resolves the issue, but it shouldn't be necessary, as Ansible-Core should already include it.

<html>
<body>
<html><head></head><body>
<hr>
<h3>Installed Packages:</h3>

Package | Version
-- | --
ansible | 9.4.0
ansible-core | 2.16.12
boto3 | 1.35.49
botocore | 1.35.49
certifi | 2024.8.30
cffi | 1.17.1
charset-normalizer | 3.4.0
cryptography | 43.0.3
idna | 3.10
Jinja2 | 3.1.4
jmespath | 1.0.1
MarkupSafe | 3.0.2
packaging | 23.2
pip | 24.3.1
pycparser | 2.22
pyparsing | 3.1.1
pyspnego | 0.11.1
python-dateutil | 2.9.0.post0
pywinrm | 0.5.0
PyYAML | 6.0.2
requests | 2.32.3
requests_ntlm | 1.3.0
resolvelib | 1.0.1
s3transfer | 0.10.3
setuptools | 65.5.0
six | 1.16.0
urllib3 | 2.2.3
xmltodict | 0.14.2

</body></html>
</body>
</html>

**Comment by teddue** [comment]
The problem persists with docker image v2.13.0

**Comment by marek1712** [comment]
v2.17.36 and still having issues.
